experimental/gw-prevent-duplicate-drop-down-selections.js

```diff
@@ -23,7 +23,6 @@ jQuery(".gfield_ddselect").change(function () {
 	jQuery(".gfield_ddselect").each(function (a) {
 		var parent_el = jQuery(this);
 		jQuery("option", jQuery(this)).each(function (b) {
-			jQuery(this).prop("disabled", false);
```

Comment:
I think removing this would likely break some existing functionality.

I'd just check for the `gpi-disabled` class on `this` and if it does have the class, then do not remove the disabled prop.

Reply:
@saifsultanc Just confirming Clay's suspicion that this breaks the core functionality of this snippet: https://www.loom.com/share/5fafb1774df342049639cb5e76e114d5